### ISSUE TITLE
Better form processing errors handling

### ISSUE THREAD

**Issue by pboguslawski** [getgrav/grav-plugin-form#567]
Grav throws ugly server error when unable to send form content using email process action (i.e. smtp server is temporarily unavailable and we don't want to use queue mode).

Would be nicer to return user to filled form with configurable message i.e. "Error sending form; please try later." and allow user to resend it.

**Comment by sridharkalaibala** [comment]
I checked the current code with an injected Symfony TransportException representing an unavailable SMTP server. Using Grav 2.1.2, Form at b6b524e and Email at 6cc50fe, the real Email::send() and Form::post() path returns a form error, retains the submitted textarea value, stops the later success-message action, and releases the failed form's recorded ID. A successful-transport control reaches the success message. The fixture uses a controlled transport and no network; it does not reproduce an actual SMTP outage or a browser resubmission.

The current Email plugin catches transport exceptions and returns a failed-send status; its form handler then fires onFormValidationError and stops processing. The friendly/configurable-message part of this request is still distinct: the current form receives the transport's error text, rather than a configurable “please try later” message.

Could you share the current Grav/Form/Email versions and exception class if the server-error behavior still occurs? Message construction or another process action can fail before this transport handler, so I would not treat every processing error as resolved by this check.

**Comment by rhukster** [comment]
Agreed, and thanks for the patience — this one's been open a while.

Most of what you asked for is already in place: when a send fails the form comes back with an error, the submitted values are kept, the success message is suppressed and the form can be resubmitted. What was missing is the part you led with, the message itself. It was passing the mail transport's own error text straight through to the page, which is both unhelpful to a visitor and more than they should see — an SMTP refusal routinely names the server, the account, and why the credentials were rejected.

That's now split. The transport's message goes to the log, where it's useful to you, and the visitor gets a plain "Your message could not be sent right now. Please try again in a few minutes." You can override the wording per email action with `error_message:`, or site-wide in the Email plugin settings. With the debugger enabled the transport's own text is appended inline, since at that point the person looking at the form is the person configuring it.

Fixed on develop in the Email plugin, shipping in **5.2.1**.
